core/camel-core-engine/src/main/docs/modules/eips/pages/dynamic-router.adoc

```diff
@@ -1,198 +0,0 @@
-[[DynamicRouter-DynamicRouter]]
```

Comment:
Why removing this?

Reply:
I have removed the whole file as `dynamic-router.adoc` as well as `dynamicRouter-eip.adoc` encompasses the same information. The same deal goes along with `request-reply.adoc` and `requestReply-eip.adoc`.